src/main/webapp/app/pages/companionDiagnosticDevicesPage/companionDiagnosticDevicePage.tsx

```diff
@@ -507,14 +501,16 @@ const CompanionDiagnosticDevicePage: React.FunctionComponent<{}> = () => {
         <Col className={classnames(...COMPONENT_PADDING)} lg={2} md={6} xs={12}>
           <Select
             placeholder={'Gene(s)'}
-            options={_.chain(parsedCDx)
-              .map(cdx => ({
-                label: cdx.biomarkerAssociation.gene,
-                value: cdx.biomarkerAssociation.gene,
-              }))
-              .uniqBy('label')
-              .sortBy('label')
-              .value()}
+            options={sortByKey(
```

Comment:
If possible, I'd like to do some chaining in the future too.

Reply:
You may be able to [do that more easily](https://github.com/tc39/proposal-pipeline-operator) in the future.